### DISCUSSION TITLE
漫游功能的具体使用

### DISCUSSION THREAD

**Discussion by PurBlu** [Richasy/Bili.Uwp#1180] [问答]
![image](https://user-images.githubusercontent.com/74239692/167812144-cb4b5cd8-a7cd-495c-a377-5019d440f43a.png)
![image](https://user-images.githubusercontent.com/74239692/167812243-d64ed231-73ab-41b5-bcca-e3cddb7895cb.png)
使用同样的代理服务器 但是无法解析成功

_另外闲碎一句 虽然似乎不符合WinUI 3的规范 但是多标签页的使用体验与效率个人认为似乎是更优解_

**Comment by KremeCN** [comment]
哔哩的解析服务器与哔哩漫游不能通用。
